packages/camera/camera_android_camerax/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_android_camerax
 description: Android implementation of the camera plugin using the CameraX library.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_android_camerax
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.6.14
+version: 0.6.14+1
```

Comment:
You might wonder why this plugin didnt need to update to 3.27. It is because it was already updated.